layers/meta-resin-beaglebone/recipes-bsp/u-boot/u-boot/0001-beaglebone-black-Use-Balena-vars-for-mmc-boot.patch

```diff
@@ -0,0 +1,76 @@
+From f5ca593b0154769ccffcbbbf79ecfb1a0757b57f Mon Sep 17 00:00:00 2001
+From: Alexandru Costache <alexandru@balena.io>
+Date: Sun, 23 Dec 2018 18:00:21 +0100
+Subject: [PATCH] beaglebone-black: Use Balena vars for mmc boot
+
+Set up necessary environment vars accordingly to enable
+booting Balena OS from SD/EMMC.
+
+Upstream-status: Inappropriate [configuration]
+Signed-off-by: Alexandru Costache <alexandru@balena.io>
+---
+ include/configs/am335x_evm.h | 8 +++-----
+ include/environment/ti/mmc.h | 6 +-----
+ 2 files changed, 4 insertions(+), 10 deletions(-)
+
+diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
+index e841b6e..0ef9fe7 100644
+--- a/include/configs/am335x_evm.h
++++ b/include/configs/am335x_evm.h
+@@ -64,7 +64,7 @@
+ 	"gpio set 53; " \
+ 	"setenv devtype mmc; " \
+ 	"setenv mmcdev " #instance"; "\
+-	"setenv bootpart " #instance":1 ; "\
++	"setenv bootpart " #instance":${resin_root_part}; "\
+ 	"run boot\0"
+ 
+ #define BOOTENV_DEV_NAME_LEGACY_MMC(devtypeu, devtypel, instance) \
+@@ -94,8 +94,6 @@
+ 	func(LEGACY_MMC, legacy_mmc, 0) \
+ 	func(MMC, mmc, 1) \
+ 	func(LEGACY_MMC, legacy_mmc, 1) \
+-	BOOT_TARGET_PXE(func) \
+-	BOOT_TARGET_DHCP(func)
+ 
+ #include <config_distro_bootcmd.h>
+ 
+@@ -107,8 +105,9 @@
+ 	DEFAULT_LINUX_BOOT_ENV \
+ 	DEFAULT_MMC_TI_ARGS \
+ 	DEFAULT_FIT_TI_ARGS \
+-	"bootpart=0:2\0" \
++	"bootpart=${resin_dev_index}:${resin_root_part}\0" \
+ 	"bootdir=/boot\0" \
++        "fdtdir=/boot\0" \
+ 	"bootfile=zImage\0" \
+ 	"board_eeprom_header=undefined\0" \
+ 	"fdtfile=undefined\0" \
+@@ -232,7 +231,6 @@
+ 		"else " \
+ 			"setenv console ttyO0,115200n8;" \
+ 		"fi;\0" \
+-	EEWIKI_NFS \
```

Comment:
Will it try NFS boot because of this?

Reply:
Not necessary anymore, I removed obsolete changes and force pushed this commit again.